pkg/monitor/operator.go

```diff
@@ -11,12 +11,30 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/cache"
 
 	configv1 "github.com/openshift/api/config/v1"
 	configclientset "github.com/openshift/client-go/config/clientset/versioned"
 )
 
+func isClusterOperatorPresent(client kubernetes.Interface) (bool, error) {
```

Comment:
Any chance moving `DoesApiResourceExist` some place else where it can be used in both the tests and in here?